newm-server/src/main/kotlin/io/newm/server/typealiases/TypeAliases.kt

```diff
@@ -2,6 +2,8 @@ package io.newm.server.typealiases
 
 import java.util.UUID
 
+typealias ListPair<T> = Pair<List<T>?, List<T>?>
```

Comment:
rename `ListPair` -> `FilterCriteria`

Reply:
I'd probably make this an actual generic type instead of a typealias so that the downstream code is more readable instead of having `first` and `second` everywhere.

Something like...

```
class <T> FilterCriteria(val includes: List<T>?, val excludes: List<T>?)
```

Reply:
That's a good idea.